com.unity.render-pipelines.universal/Editor/Lighting/UniversalRenderPipelineLightUI.PresetInspector.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    using CED = CoreEditorDrawer<UniversalRenderPipelineSerializedLight>;
+
+    internal partial class UniversalRenderPipelineLightUI
```

Comment:
internal is already default here. Not needed. (Same for other partial definition of this one)

Reply:
yes I agree, this is for consistency with the other partial declarations as suggested in the above conversations.

Reply:
No I mean, the other also don't need internal as it is the default one already applied